en/identity-server/next/docs/apis/organization-apis/restapis/configs.yaml

```diff
@@ -0,0 +1,513 @@
+openapi: 3.0.0
+info:
+  version: "v1"
+  title: 'Configurations API'
+  description: |
+    This document specifies the **Server related Configurations RESTful API** of organizations in **WSO2 Identity Server**.
+security:
+  - OAuth2: []
+  - BasicAuth: []
```

Comment:
BasicAuth support is not available for sub org APIs.
Let's remove . 
Check other organizational APIs as well